test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/HtmlTagsTest/VoidElementFollowedByCloseTag.stree.txt

```diff
@@ -1,563 +1,612 @@
-            SyntaxKind.NewLine;[LF];
-        MetaCode span - Gen<None> - [}] - SpanEditHandler;Accepts:None - (547:44,0) - Tokens:1
-            SyntaxKind.RightBrace;[}];
-    Markup span - Gen<None> - [LF] - SpanEditHandler;Accepts:Any - (548:44,1) - Tokens:1
-        SyntaxKind.NewLine;[LF];
-    Statement block - Gen<None> - 32 - (550:45,0)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (550:45,0) - Tokens:1
-            SyntaxKind.Transition;[@];
-        MetaCode span - Gen<None> - [{] - SpanEditHandler;Accepts:None - (551:45,1) - Tokens:1
-            SyntaxKind.LeftBrace;[{];
-        Code span - Gen<Stmt> - [LF] - AutoCompleteEditHandler;Accepts:Any,AutoComplete:[<null>];AtEOL - (552:45,2) - Tokens:1
-            SyntaxKind.NewLine;[LF];
-        Markup block - Gen<None> - 12 - (554:46,0)
-            Tag block - Gen<None> - 5 - (554:46,0)
-                Markup span - Gen<Markup> - [<wbr>] - SpanEditHandler;Accepts:None - (554:46,0) - Tokens:3
-                    SyntaxKind.OpenAngle;[<];
-                    SyntaxKind.Text;[wbr];
-                    SyntaxKind.CloseAngle;[>];
-            Markup span - Gen<Markup> - [ ] - SpanEditHandler;Accepts:Any - (559:46,5) - Tokens:1
-                SyntaxKind.Whitespace;[ ];
-            Tag block - Gen<None> - 6 - (560:46,6)
-                Markup span - Gen<Markup> - [</wbr>] - SpanEditHandler;Accepts:None - (560:46,6) - Tokens:4
-                    SyntaxKind.OpenAngle;[<];
-                    SyntaxKind.ForwardSlash;[/];
-                    SyntaxKind.Text;[wbr];
-                    SyntaxKind.CloseAngle;[>];
-        Code span - Gen<Stmt> - [var x = true;LF] - SpanEditHandler;Accepts:Any - (566:46,12) - Tokens:9
-            SyntaxKind.Identifier;[var];
-            SyntaxKind.Whitespace;[ ];
-            SyntaxKind.Identifier;[x];
-            SyntaxKind.Whitespace;[ ];
-            SyntaxKind.Assign;[=];
-            SyntaxKind.Whitespace;[ ];
-            SyntaxKind.Keyword;[true];
-            SyntaxKind.Semicolon;[;];
-            SyntaxKind.NewLine;[LF];
-        MetaCode span - Gen<None> - [}] - SpanEditHandler;Accepts:None - (581:47,0) - Tokens:1
-            SyntaxKind.RightBrace;[}];
-    Markup span - Gen<None> - [LF] - SpanEditHandler;Accepts:Any - (582:47,1) - Tokens:1
-        SyntaxKind.NewLine;[LF];
+RazorDocument - [0..584)::584 - [@{LF<area> </area>var x = true;LF}LF@{LF<base> </base>var x = true;LF}LF@{LF<br> </br>var x = true;LF}LF@{LF<col> </col>var x = true;LF}LF@{LF<command> </command>var x = true;LF}LF@{LF<embed> </embed>var x = true;LF}LF@{LF<hr> </hr>var x = true;LF}LF@{LF<img> </img>var x = true;LF}LF@{LF<input> </input>var x = true;LF}LF@{LF<keygen> </keygen>var x = true;LF}LF@{LF<link> </link>var x = true;LF}LF@{LF<meta> </meta>var x = true;LF}LF@{LF<param> </param>var x = true;LF}LF@{LF<source> </source>var x = true;LF}LF@{LF<track> </track>var x = true;LF}LF@{LF<wbr> </wbr>var x = true;LF}LF]
```

Comment:
This looks huge but it's actually totally repetitive. Can just review the first repetition and the very end.